### PR TITLE
DDF-1963: added tables to managing app contents

### PR DESCRIPTION
**Pull request by ricklarsen**
#### What does this PR do?

Cleans up structure of managing application sections. 
Separates configuration tables into component files included where needed.

#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?

@Lambeaux @jrnorth @mcalcote 
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).

@shaundmorris
@stustison
#### How should this be tested?

- successful build
- added documentation correct and formatted.

#### What are the relevant tickets?

https://codice.atlassian.net/browse/DDF-1963

#### Checklist:
- [x] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

File: catalog/core/catalog-core-localstorageprovider/src/main/resources/OSGI-INF/metatype/metatype.xml [hunk 1]
```diff
@@ -18,7 +18,7 @@
     <OCD name="Content File System Storage Provider"
          id="org.codice.ddf.catalog.content.impl.FileSystemStorageProvider">
         <AD
-                description="Specifies the directory to use for the content repository. A shutdown of the server is necessary for this property to take effect. If a filepath is provided with directories that don't exist, File System Provider will attempt to create them. Out of the box (without configuration), the content repository is <INSTALL_DIR>/content/store."
```
Comment (shaundmorris) [inline catalog/core/catalog-core-localstorageprovider/src/main/resources/OSGI-INF/metatype/metatype.xml]: Why did you have to switch out `<` for `&lt`? Was it a problem rendering or did it cause an issue with generating the tables?

Comment (ricklarsen) [inline catalog/core/catalog-core-localstorageprovider/src/main/resources/OSGI-INF/metatype/metatype.xml]: it was an issue with generating the tables. It called it "improperly formatted xml"

**Comment by shaundmorris** [comment]
@ricklarsen there are merge conflicts on this PR

**Comment by ricklarsen** [comment]
fixed merge conflicts

**Comment by shaundmorris** [comment]
Looks like the bamboo build passed but ran out of memory

**Comment by shaundmorris** [comment]
👍

**Comment by jrnorth** [comment]
LGTM

**Comment by stustison** [comment]
Way too many changes in here for me to reliably review this. I'd suggest just doing sections at a time next time. I skimmed it and it seems alright.

**Comment by ricklarsen** [comment]
Should I break it up?

**Comment by shaundmorris** [comment]
@ricklarsen breaking it up would help reviewers with doing more in-depth, quality reviews. I did build the PR and clicked through the docs, so it seemed okay. But I didn't do the best review on all of the content.

**Comment by ricklarsen** [comment]
I'll break it up and reopen.

**Pull request closed without merging**